requirements.txt

```diff
@@ -3,4 +3,5 @@
 nose
 rednose
 coverage
+smart-sentinel>=0.2.0
```

Comment:
The [Travis build](https://travis-ci.org/rfloriano/smart-sentinel) for this lib is failing, is it ok to use?